bcipy/signal/process/extract_gaze.py

```diff
@@ -29,20 +29,45 @@ def extract_eye_info(data):
     left_eye = np.vstack((np.array(lx), np.array(ly))).T
     right_eye = np.vstack((np.array(rx), np.array(ry))).T
 
-    # Remove ALL blinks (i.e. Nan values) regardless of which eye it occurs.
-    # Make sure that the number of samples are the same for both eyes
+    # Remove all blinks (i.e. Nan values) regardless of which eye it occurs.
```

Comment:
Remove comments in this file. Is the blink / NaN transform still needed, though?

Reply:
NaN transform is unfortunately needed for one of the models. The goal is to keep the inquiry length constant across all inquiries during an experiment. I'm thinking of updating the return statement with a flag to remove an inquiry if it includes too many NaN value though, because I have seen a few extreme cases that the inquiry was basically unusable. I will update this script later. I deleted all the comments.